string stl_path
string entity_name
---
bool success
string message
